agenda-services/src/main/java/org/exoplatform/agenda/util/AgendaDateUtils.java

```diff
@@ -133,5 +135,9 @@ public static ZonedDateTime buildOccurrenceDateTime(String occurrenceId) {
     }
     return ZonedDateTime.parse(occurrenceId, OCCURRENCE_ID_FORMATTER);
   }
+  
+  public static String formatUpdatedDate(ZonedDateTime zonedDateTime) {
```

Comment:
if this is an operation that will be used to format the updated time only, it's better to not place it here.
IMO, the method name has to reflect what it does as processing and not define were it will be used.

Reply:
ok i will remove it ti NotificationUtils and change name to formatNotificationUpdatedDate